Simplenote/src/main/java/com/automattic/simplenote/models/Note.java

```diff
@@ -99,7 +99,7 @@ public static Query<Note> search(Bucket<Note> noteBucket, String searchString) {
     public static Query<Note> allInTag(Bucket<Note> noteBucket, String tag) {
         return noteBucket.query()
                 .where(DELETED_PROPERTY, ComparisonType.NOT_EQUAL_TO, true)
-                .where(TAGS_PROPERTY, ComparisonType.LIKE, tag);
+                .where(TAGS_PROPERTY, ComparisonType.EQUAL_TO, tag);
```

Comment:
The app is creating an index of all of the tags associated with a note:

<img width="1162" alt="Screenshot 2024-01-23 at 5 26 50 PM" src="https://github.com/Automattic/simplenote-android/assets/789137/b64f869a-ccf4-4d63-9e26-6131b0f15d60">

The query is left joining on that index, something like: 

```
SELECT count(objects.rowid) as `total` FROM `objects` 
LEFT JOIN indexes AS i0 ON objects.bucket = i0.bucket AND objects.key = i0.key AND i0.name="deleted"
LEFT JOIN indexes AS i1 ON objects.bucket = i1.bucket AND objects.key = i1.key AND i1.name="tags"
WHERE objects.bucket = "note" AND (  i0.value IS NULL OR i0.value !=  1) AND (  i1.value IS NOT NULL AND i1.value = "todo") 
```

So I think we're cool to do an equal comparison here! Search still works because it appends a like on a different query.

Reply:
oh wow. thanks for clarifying.

I see multiple "opportunities for improvement" here 🙃 

let's try a quick safety-check:
 - search for `buecher` for the tag `bücher`
 - search for `Dog` for the tag `dog`
 - search for `brau` for `bräu`

I'm not familiar with the querying (you might be) and so thought we should verify that there's no language-sensitive behavior incorporated into `LIKE`

otherwise, totally :shipit:

Reply:
Search is using a different query for tag suggestions with a `LIKE`, and those searches don't bring up matches :|(except for the `Dog` one as it doesn't seem to be case sensitive)

We'll have to poke at that to see if we can fix that or if it's a limitation of SQLite 3